### PR TITLE
Move chunking logic to embedding service + allow user to edit elements

### PR DESCRIPTION
**Pull request by Navidur1**

### CODE REVIEW

File: ocr_flow.py [hunk 1]
```diff
@@ -137,3 +57,44 @@ def upload_pdf():
         return jsonify({'gcs_pdf_url': gcs_pdf_url, 'file_id': str(file_id), 'ocr_result': ocr_results}), 200
     
     return jsonify({'gcs_pdf_url': "fake"}), 400
+
+def ocr_flow(uploaded_file, file_id):
```
Comment (Ismail-Ben) [inline ocr_flow.py:61]: whats the diff between this and the old ocr_flow function you removed

**Review by Ismail-Ben — COMMENTED** [review]
Looks good but have a question.  To confirm what I understood, ocr service is just a wrapper for the ocr api that stores the outputted elements from unstructured and displays then to the frontend. Then after confirming results on front end wouldn't we need to update the base elements before chunking? Is this included in the chunking function L#68.

File: embedding_service.py [hunk 2]
```diff
@@ -48,4 +56,48 @@ def create_embeddings():
...
+    threshold = 512
+    stored_ids = []
+    cur_element_ids = []
+    cur_text = ""
+    chunk_success = True
+
+    for i in range(len(elements)):
+        # Update element in mongoDB
+        data = elements[i]
+        chunk_success, error_message = update_one('Elements', {'_id': ObjectId(data['id'])}, {'$set': {'text': data['text']}})
```
Comment (Ismail-Ben) [inline embedding_service.py:70]: chunk_success misleading variable name

File: ocr_flow.py (hunk 1 above)
Comment (Navidur1) [reply ocr_flow.py:61]: Same thing except it doesn't call `chunk_elements` and I moved it down in the file

**Comment by Navidur1** [comment]
> Looks good but have a question. To confirm what I understood, ocr service is just a wrapper for the ocr api that stores the outputted elements from unstructured and displays then to the frontend. Then after confirming results on front end wouldn't we need to update the base elements before chunking? Is this included in the chunking function L#68.

Yes the elements get updated in the chunking function based on the user's edits

**Review by Ismail-Ben — APPROVED** [review]

**Pull request merged**